modules/edit-modes/src/lib/geojson-edit-mode.ts

```diff
@@ -259,17 +259,7 @@ export class GeoJsonEditMode implements EditMode<FeatureCollection, GuideFeature
   handleStopDragging(event: StopDraggingEvent, props: ModeProps<FeatureCollection>): void {}
   handleDragging(event: DraggingEvent, props: ModeProps<FeatureCollection>): void {}
 
-  handleKeyUp(event: KeyboardEvent, props: ModeProps<FeatureCollection>): void {
-    if (event.key === 'Escape') {
-      this.resetClickSequence();
-      props.onEdit({
-        // Because the new drawing feature is dropped, so the data will keep as the same.
-        updatedData: props.data,
-        editType: 'cancelFeature',
-        editContext: {},
-      });
-    }
-  }
+  handleKeyUp(event: KeyboardEvent, props: ModeProps<FeatureCollection>): void {}
```

Comment:
Restore this file. By removing this you are breaking Esc key event for many extended GeoJsonEdit modes.

Reply:
Never mind, I'll do it.